README.md

```diff
@@ -118,18 +121,20 @@ target in the Unity editor.
 
 ## Usage
 
-   1. Add the unitypackage to your plugin project (assuming you are developing a
-      plugin).  Please read the [Plugin Redistribution](#plugin-redistribution)
-      section when you're considering how to export your package to share with
-      your users.
+   1. Add the `play-services-resolver-1.2.75.0.unitypackage` to your plugin 
```

Comment:
Simply adding the plugin to your project isn't recommend as the Version Handler component will activate / enable all subcomponents of the plugin, preventing resolution of conflicting versions of this plugin.  This still needs a reference to the Plugin Redistribution section to ensure developers import the plugin correctly when redistributing the plugin.  I see Plugin Redistribution has been moved to bullet 3, would it make sense to add a note that step 3 is really important?